include/courtroom.h

```diff
@@ -225,8 +225,11 @@ class Courtroom : public QMainWindow
   // handle server-side clock animation and display
   void handle_clock(QString time);
 
-  // handle request to change theme variant
-  void handle_theme_variant(QString theme_variant);
+  // handle request to change gamemode
```

Comment:
The comments here and above the function below could be made into documentation.

Reply:
See #dev